frameworks/zsh4humans.zsh

```diff
@@ -0,0 +1,24 @@
+(
+
+emulate -L zsh -o err_return -o no_unset -o no_aliases
+cd -- $1
+# download zsh4humans v5
+curl -fsSLO https://github.com/romkatv/zsh4humans/archive/refs/heads/v5.tar.gz
+# copy over .zshenv and .zshrc
+tar -xzf v5.tar.gz
+cp zsh4humans-5/{.zshenv,.zshrc} ./
+rm -r v5.tar.gz zsh4humans-5
+# create a config for powerlevel10k
+>.p10k.zsh <<<$'POWERLEVEL9K_MODE=ascii\nPOWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true'
+# initialize zsh4humans
+export HOME=$PWD
+(( $+commands[script] )) && {
+  # use script if available
+  script -qec "zsh -is </dev/null" /dev/null
```

Comment:
This failed with `script: illegal option -- e` on macOS.

And why not simply use 
```
if condition; then
  do-this
else
  do-that
fi
```
instead of
```
condition && {
  do-this
  return
} || {
  do-that
}
```
?

Reply:
I've removed the call to `script` to resolve both of your comments. (The `&&` and `||` were used as a workaround for bugs in zsh 5.4.1 that trigger when `err_return` is in effect.)

Reply:
Cool, good to know about the workaround.